examples/dataform_json_checks/invalid_defaultschema/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//tests:__subpackages__"])
+
+load("//tools:node_modules.bzl", "node_modules")
+
+filegroup(
+    name = "files",
+    srcs = glob([
+        "**/*.*",
+    ]),
+)
+
+node_modules(
+    name = "node_modules",
+    deps = [
+        "//core:package_tar",
+        "//crossdb:package_tar",
```

Comment:
none of your code actually depends on this, remove it from each `BUILD` file?

Reply:
Can't delete it. See comments above. Same for all similar ones.

Reply:
If I delete that I would get this error:

```
ERROR: /Users/cesc/dataform/tests/api/BUILD:29:1: no such target '//examples/dataform_json_checks/invalid
_warehouse:node_modules': target 'node_modules' not declared in package 'examples/dataform_json_checks/in
valid_warehouse' defined by /Users/cesc/dataform/examples/dataform_json_checks/invalid_warehouse/BUILD and referenced by '//tests/api:api_bin'
```

By the way, have you seen the changes in `tests/api/BUILD` file ?

Reply:
I think we had a bit of confusion here! I'm not saying we can get rid of these `BUILD` files (except `examples/dataform_json_checks/BUILD` which you have already removed), but instead that you can get rid of this dependency on `//crossdb:package_tar` (and `//protos:package_tar`, I believe).